app/jobs/retrieve_award_job.rb

```diff
@@ -13,6 +14,7 @@ def perform(trainee)
       trainee.award!
     elsif continue_polling?(trainee)
       self.class.set(wait: Settings.jobs.poll_delay_hours.hours).perform_later(trainee)
+    elsif send_message_to_slack(trainee, self.class)
```

Comment:
This should be an `else`.
`elsif` is conditional. In this case we just want to run the code after all the other `if/elsif` has failed.

Also best to send `self.class.name`, which is a string.

Reply:
cheers, I've fixed